autoflip_calculators/text_detection_calculator.proto

```diff
@@ -32,4 +32,16 @@ message TextDetectionCalculatorOptions {
   // If true, generate a score from the appearance of the text and use it to
   // modulate the detection scores for text bboxes.
   optional bool use_visual_scorer = 2 [default = true];
+
+  // Path to the text detection TF model (ex: /path/to/modelname.pb).
+  //TODO(zzhencchen): support relative paths.
+  optional string model_path = 3 [default = "/usr/local/google/home/zzhencchen/mediapipe/mediapipe/models/frozen_east_text_detection.pb"];
```

Comment:
Probably best not to put a default path because it won't work on other computers.  Just remove the default, maybe add a comment with the typical location.